nucypher/cli/commands/worklock.py

```diff
@@ -258,19 +261,69 @@ def refund(general_config, worklock_options, registry_options, force, hw_wallet)
 
 
 @worklock.command()
-@group_registry_options
 @group_general_config
-@option_checksum_address
-def burn_unclaimed_tokens(general_config, registry_options, checksum_address):
+@group_registry_options
+@group_worklock_options
+@option_force
+@option_hw_wallet
+@click.option('--gas-limit', help="Gas limit per each verification transaction", type=click.IntRange(min=60000))
+# TODO: Consider moving to administrator (nucypher-deploy)
+def post_init(general_config, registry_options, worklock_options, force, hw_wallet, gas_limit):
```

Comment:
Could confuse, because we have `enable_claiming` in `Bidder` and `WorkLockAgent`.
Also I like how it looks now `nucypher worklock post-init`, but I'm open to renamings

Reply:
👌 I was sure that there was a legitimate reason 😄 .

I'd vote for a more meaningful name, but interested in other people's opinions as well. We could do something like `close-bidding` or `end-bidding` or `start-claiming-period` or ... (?)

One question, this method can only be called post-cancellation end date, correct? Do we need to check that cancellation date is surpassed or is this check done deeper down in one of our underlying function calls?

Reply:
There is check  `self._ensure_cancellation_window(ensure_closed=True)` In `force_refund()` and in `verify_bidding_correctness()`